be/src/Project/Web/Seller/Seller.Web/Areas/Clients/ApiControllers/ClientNotificationTypeApi.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Foundation.ApiExtensions.Definitions;
+using Foundation.Localization;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Localization;
+using Seller.Web.Areas.Clients.ApiRequestModels;
+using Seller.Web.Areas.Clients.DomainModels;
+using Seller.Web.Areas.Clients.Repositories.NotificationTypes;
+using System;
+using System.Globalization;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Seller.Web.Areas.Clients.ApiControllers
+{
+    [Area("Clients")]
+    public class ClientNotificationTypeApiController : ControllerBase
```

Comment:
**suggestion** Should be ClientNotificationTypesApiController instead of ClientNotificationTypeApiController